2024-02-13 16:12:14,902 INFO    MainThread:31716 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-13 16:12:14,902 INFO    MainThread:31716 [wandb_setup.py:_flush():76] Configure stats pid to 31716
2024-02-13 16:12:14,903 INFO    MainThread:31716 [wandb_setup.py:_flush():76] Loading settings from C:\Users\A114424304\.config\wandb\settings
2024-02-13 16:12:14,903 INFO    MainThread:31716 [wandb_setup.py:_flush():76] Loading settings from C:\workspace\udacity_training\wandb\settings
2024-02-13 16:12:14,903 INFO    MainThread:31716 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-13 16:12:14,903 INFO    MainThread:31716 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-13 16:12:14,904 INFO    MainThread:31716 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'test_wandb.py', 'program_abspath': 'c:\\workspace\\udacity_training\\test_wandb.py', 'program': 'c:/workspace/udacity_training/test_wandb.py'}
2024-02-13 16:12:14,904 INFO    MainThread:31716 [wandb_init.py:_log_setup():526] Logging user logs to C:\workspace\udacity_training\wandb\run-20240213_161214-pfj595og\logs\debug.log
2024-02-13 16:12:14,904 INFO    MainThread:31716 [wandb_init.py:_log_setup():527] Logging internal logs to C:\workspace\udacity_training\wandb\run-20240213_161214-pfj595og\logs\debug-internal.log
2024-02-13 16:12:14,905 INFO    MainThread:31716 [wandb_init.py:init():566] calling init triggers
2024-02-13 16:12:14,905 INFO    MainThread:31716 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.02, 'architecture': 'CNN', 'dataset': 'CIFAR-100', 'epochs': 10}
2024-02-13 16:12:14,905 INFO    MainThread:31716 [wandb_init.py:init():616] starting backend
2024-02-13 16:12:14,905 INFO    MainThread:31716 [wandb_init.py:init():620] setting up manager
2024-02-13 16:12:14,909 INFO    MainThread:31716 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-02-13 16:12:14,911 INFO    MainThread:31716 [wandb_init.py:init():628] backend started and connected
2024-02-13 16:12:14,915 INFO    MainThread:31716 [wandb_init.py:init():720] updated telemetry
2024-02-13 16:12:14,915 INFO    MainThread:31716 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-13 16:13:10,320 WARNING MainThread:31716 [wandb_init.py:init():1198] interrupted
Traceback (most recent call last):
  File "C:\Anaconda\envs\udaproject\lib\site-packages\wandb\sdk\wandb_init.py", line 1176, in init
    run = wi.init()
  File "C:\Anaconda\envs\udaproject\lib\site-packages\wandb\sdk\wandb_init.py", line 756, in init
    result = run_init_handle.wait(
  File "C:\Anaconda\envs\udaproject\lib\site-packages\wandb\sdk\lib\mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "C:\Anaconda\envs\udaproject\lib\site-packages\wandb\sdk\lib\mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "C:\Anaconda\envs\udaproject\lib\site-packages\wandb\sdk\lib\mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "C:\Anaconda\envs\udaproject\lib\threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "C:\Anaconda\envs\udaproject\lib\threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
